### ISSUE TITLE
Problem Memory Size parametres

### ISSUE THREAD

**Issue by ToniBC** [schellingb/dosbox-pure#122]
Hello, there is a small problem in the configuration with the parameter "Memory Size", because you can select "not to use xms / ems" but it would have to be separate.

Let me explain better, there are several games, such as SideLine, that require the following.
**xms = false
ems = false
umb = false**

But you also need.
**memsize = 16**

So in Dosbox Pure we can put different memory sizes, but if we select "No xms / ems" it stays as memsize = 0 and the game does not start because it gives a memory error.

If we put 16 memory it starts up, but it stays black because xms / ems are activated.

In summary, disabling those three parameters above are different options than the memory size we assign to it and it would have to be a separate option.

I get the game to work, for example, if I load it with the value of memsize = 16 in the executable selection screen, then, before I run the .exe, I go to the menu, I deactivate xms / ems and I start the game and without problems, but the second time you try to put it, as it has memsize disabled, it gives an error.

Thanks.

**Comment by bigroy00** [comment]
How do you turn off ems (i.e. ems = false)? I cant see it anywhere in Options
Need this for Ultima VII Part 1 - The Black Gate + Forge of Virtue (1992)

**Comment by schellingb** [comment]
Interesting, I didn't know such games exist.
Do you think there's any potential for problems to just default to 16MB when the option "Disable extended memory (no EMS/XMS)" is selected? As always I try to limit the amount of options so still hoping to avoid the need for multiple separate memory options.
Thanks for reporting this!

**Comment by schellingb** [comment]
I went ahead and put it in because it's a one line change. You can try version 0.11 and see if it solves the issue for you.

**Comment by bigroy00** [comment]
I tested version 0.11 on 3 games that require ems to be turned off and they all work fine
Out of This World (1991) aka Another World
Ultima VII Part 1 - The Black Gate + Forge of Virtue (1992)
Ultima VII Part 2 - Serpent Isle + The Silver Seed (1993)
Thanks

**Comment by ToniBC** [comment]
Yes, I think leaving 16 Mb by default when xms / ems memory is disabled would be enough. Then I test it to confirm it.

**Comment by ToniBC** [comment]
It seems to work now all well, for the moment with these games, we leave the subject closed.

Thanks a lot.

**Comment by GoukiKain** [comment]
Hello, just wanted to report that this method does not work for SideLine: the game loads but stays on a black screen after the intro while the demo sounds can still be heard in the background. This happens by selecting either "Disable extended memory (no EMS/XMS)" or "memory size = 16".
SideLine is one of the best shooter (if not THE best) for DOS and very under appreciated.

Out of this World and Ultima games work fine when "Disable extended memory (no EMS/XMS)" is selected.

**Comment by GoukiKain** [comment]
OK for anyone struggling with it I managed to get SideLine to work.
In addition to you need to load a memory manager for DOS before the executable. I used JEMMEX (https://www.japheth.de/Jemm.html) with command line JEMMEX NOEMS LOAD. Found the solution on Vogons (https://www.vogons.org/viewtopic.php?t=33420).
Cheers!
